cypress/e2e/gui/gui-engage-sphere.spec.cy.js

```diff
@@ -0,0 +1,190 @@
+const API_URL = Cypress.env('API_URL');
+const CUSTOMERS_API_URL = `${API_URL}/customers`;
+describe('Customers - filters and others ', () => {
+    beforeEach(() => {
+        cy.clearCookies();
+        cy.setCookie('cookieConsent', 'accepted');
+        cy.visit('/');
+        cy.contains('h2', 'Hi there!', { timeout: 5000 })
+    });
+
+    it('keeps the filters when coming back from the customer details view', () => {
+        const clientFilterData = {
+            size: 'Small',
+            industry: 'Technology'
+        };
+        cy.intercept('GET', `${CUSTOMERS_API_URL}?page=1&limit=10&size=${clientFilterData.size}&industry=${clientFilterData.industry}`,
```

Comment:
```suggestion
        cy.intercept('GET', `${CUSTOMERS_API_URL}?page=1&limit=10&size=${clientFilterData.size}&industry=${clientFilterData.industry}`,

```

Aqui uma linha em branco ajuda também, pois separa uma coisa (`intercept`) de outras `cy.get`s.